test/test_ops.py

```diff
@@ -647,6 +647,46 @@ def test_convert_boxes_to_roi_format(self):
                 self.assertTrue(torch.equal(ref_tensor, ops._utils.convert_boxes_to_roi_format(box_sequence)))
 
 
+class BoxTester(unittest.TestCase):
+    def test_bbox_xywh(self):
+        # Simple test convert boxes to xywh and back. Make sure they are same.
+        # box_tensor is in x1 y1 x2 y2 format.
+        box_tensor = torch.tensor([[0, 0, 100, 100], [0, 0, 0, 0],
+                                  [10, 15, 30, 35], [23, 35, 93, 95]], dtype=torch.float)
+        exp_xywh = torch.tensor([[0, 0, 100, 100], [0, 0, 0, 0],
+                                [10, 15, 20, 20], [23, 35, 70, 60]], dtype=torch.float)
+
+        box_xywh = ops.box_xyxy_to_xywh(box_tensor)
+        assert exp_xywh.size() == torch.Size([4, 4])
```

Comment:
nit: in this project we use `self.assertEqual` and similar methods to test things... I have no strong opinion about that, but it would be better to keep conform test code...

Reply:
K I will change the asserts for consistency 👍

Reply:
@vfdev-5 Not sure I agree here. `self.assertEqual` is the `unittest` style. AFAIK we run all our tests with `pytest` and they have [great introspection](https://docs.pytest.org/en/stable/assert.html). While the "old" style is fully compatible, I don't think we should enforce it, but rather embrace the `pytest` way.

Reply:
Let us decide and fix up this in a follow up PR. Right now tests are passing.

Reply:
This is more of a general decision and has little to do with this PR anyway.

Reply:
In a follow up PR Once we decide on what `assert` should be used, I will replace all of them in completely.

Reply:
Let's use `assertEqual`. We moved away from `assert` in the past, see https://github.com/pytorch/vision/issues/1483

Reply:
Changed to assertEqual. In a new PR I will change in other places as well.